src/MLDatasets.jl

```diff
@@ -12,6 +12,8 @@ using DelimitedFiles: readdlm
 using FileIO
 import CSV
 using LazyModules: @lazy
+using NPZ: npzread
```

Comment:
Does NPZ need to a be hard import?

Reply:
No, we can use `@ require` both for NPZ and Statistics.